hdcp-version-registry-respec.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Encrypted Media Extensions HDCP Version Registry</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
+    <script class="remove">
+    var respecConfig = {
+      // specification status (e.g. WD, LCWD, NOTE, etc.). If in doubt use ED.
+      specStatus: "DRY",
+
+      // the specification's short name, as in https://www.w3.org/TR/short-name/
+      shortName: "eme-hdcp-version-registry",
+
+      // if there a publicly available Editor's Draft, this is the link
+      edDraftURI: "https://w3c.github.io/encrypted-media/hdcp-version-registry.html",
+
+      // if this is a LCWD, uncomment and set the end of its review period
+      // lcEnd: "2009-08-05",
+
```

Comment:
We don't really do Last Call Working Drafts anymore, so no need to keep these comments around, `lcEnd` should not be of any use.

```suggestion
```

Reply:
I have removed this. Thanks!